src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -102,6 +108,21 @@ resource identityRoleAssignments 'Microsoft.Authorization/roleAssignments@2022-0
   }
 }]
 
+// Create managed identity to manage exports in cost management
```

Comment:
nit: Do we need a CM Contributor role in the list of roles? (Basing this on the comment. May also want to change the `storageRbacRoles` variable name as well.)

Reply:
To create an export the identity needs Storage Account Contributor and Reader permissions.
For ADF to read/write data to/from the storage account it needs Storage Blob Data Contributor permissions.
The identity needs to be granted EA/MCA/CM Contributor permissions against the export scope before adding it to settings.json.